### PR TITLE
feat: add output for service name

### PR DESCRIPTION
**Pull request by cyberhck**
Please confirm that you have done the following before requesting reviews:

- [x] I have confirmed that the PR type is appropriate for the change I am making according to the [Honest Pull Request and Commit Message Naming Conventions](https://www.notion.so/honestbank/Pull-Request-and-Commit-Message-Naming-Conventions-bd97f2cbb34c4c73b1ff3a3e384b850c).
- [x] I have typed an adequate description that explains **why** I am making this change.
- [x] I have installed and run standard pre-commit hooks that lints and validates my code.

### Description

add `service_name` output

- - -
This change is [Reviewable](https://reviewable.io/reviews/honestbank/terraform-pagerduty-alerting/15)

### CODE REVIEW

File: modules/pagerduty-service/outputs.tf [hunk 1]
```diff
@@ -2,3 +2,7 @@ output "id" {
   description = "The `id` attribute of the service."
   value       = pagerduty_service.service.id
 }
+
+output "service_name" {
+  value = pagerduty_service.service.name
```
Comment (jai) [inline modules/pagerduty-service/outputs.tf:7]: please add a description

File: modules/pagerduty-service/outputs.tf [hunk 2]
```diff
@@ -2,3 +2,7 @@ output "id" {
   description = "The `id` attribute of the service."
   value       = pagerduty_service.service.id
 }
+
+output "service_name" {
```
Comment (jai) [inline modules/pagerduty-service/outputs.tf:6]: please validate this by adding to https://github.com/honestbank/terraform-pagerduty-alerting/blob/main/test/pagerduty_service_test.go

**Review by jai — CHANGES_REQUESTED** [review]
thanks for the update/change!

**Review by JoseFMP — APPROVED** [review]

**Review by honestsubhadip — APPROVED** [review]

**Review by jai — APPROVED** [review]

File: modules/pagerduty-service/outputs.tf [hunk 3]
```diff
@@ -2,3 +2,8 @@ output "id" {
...
+  description = "The name of the service being created"
```
Comment (jai) [inline modules/pagerduty-service/outputs.tf:7]:
```suggestion
  description = "The name of the service being created."
```

**Pull request merged**